src/main/resources/templates/room.html

```diff
@@ -513,6 +551,25 @@ <h5>Integration with the mob tool</h5>
       prependToHistory(timerRequest);
     });
 
+    eventSource.addEventListener('GOAL_REQUEST', (event) => {
+      console.log('handling event GOAL_REQUEST ' + event.data);
+      let goalRequest = JSON.parse(event.data);
+      let requested = goalRequest["requested"];
+      let newGoal = goalRequest["goal"];
+      let user = goalRequest["user"];
+
+      if (newGoal === null && user === null) {
+        console.log('Resetting state');
+
+        goal = null;
```

Comment:
those two state changes are sort of a transaction, no? When we want the one, we want the other too.
Meant are Lines 564 and 565